src/main/java/ng/abdlquadri/eventbus/EventBusFrameHandler.java

```diff
@@ -46,7 +47,8 @@ public void run() {
 
   @Override
   public void channelInactive(ChannelHandlerContext ctx) throws Exception {
-    log.log(Level.SEVERE, "CHANNEL NOT ACTIVE");
+	// log OR throw exception, never both
```

Comment:
Yes that's always better. However I'm not sure about pull requests myself, this is the second time  I do that ;)

Actually, regarding exception handling, it might be better to use a Handler<AsyncResult<Void>> instead of a ConnectHandler.  What you do is create a AsyncResult a la vertx which has a succeeded()/failed() to fill, including an exception when it failed, and a result, which is a void in this case. The benefit is that you have an object which gives a state of success/failure and an exception. In your code there is no callback way of catching errors except a big try/catch around it. 
For the Eventbus itsself, might be an idea to add a Eventbus.registerExceptionHandler(...) ? There you can notify errors if they occured at Eventbus.java around line 250+. Just an idea...